_config.yml

```
@@ -12,7 +12,7 @@ permalink: /:categories/:title/index.html
 
 download_page_tags:
   release: "release-20210321"
-  playtest: "playtest-20230110"
+  playtest: "playtest-20230219"
```

```suggestion
  playtest: "playtest-20230218"
```

https://github.com/OpenRA/OpenRA/releases/tag/playtest-20230218